homework/05LessonFive.js

```diff
@@ -0,0 +1,140 @@
+'use strict';
+
+var assert = require('assert');
+
+// ****
+// More Operators
+// ****
+
+// DO NOT MODIFY
+var one   = 1;
+var two   = 2;
+var three = 3;
+var four  = 4;
+var five  = 5;
+var six   = 6;
+
+// Problem 1:
+// What is the result of applying the increment (++) operator to var one?
+var onePlusPlus = 2;
+
+// Problem 2:
+// What is the result of applying the decrement (--) operator to var two?
+var twoMinusMinus = 1;
+
+// Problem 3:
+// What is the result of three += 3?
+var threePlusAssignmentThree = 6;
+
+// Problem 4:
+// What is the result of four -= 3?
+var fourMinusAssignmentThree = 1;
+
+// Problem 5:
+// What is the result of five *= 2?
+var fiveMultiplicationAssignmentTwo = 10;
+
+// Problem 6:
+// What is the result of six /= 3?
+var sixDivisionAssignmentThree = 2;
+
+// ****
+// Loops
+// ****
+
+// DO NOT MODIFY
+var whileLoopArray = [];
+var whileLoopCounter = 0;
+while (whileLoopCounter < 10) {
+  whileLoopArray.push(whileLoopCounter);
+  whileLoopCounter++;
+}
+
+// Problem 7:
+// Fill in whileLoopArrayResult so that it matches whileLoopArray
+var whileLoopArrayResult = [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9 ];
+
+// DO NOT MODIFY
+var forLoopArray = [];
+var forLoopCounter;
+for (forLoopCounter = 0; forLoopCounter > -10; forLoopCounter--) {
+  forLoopArray.push(forLoopCounter);
+}
+
+// Problem 8:
+// Fill in forLoopArrayResult so that it matches forLoopArray
+var forLoopArrayResult = [ 0, -1, -2, -3, -4, -5, -6, -7, -8, -9 ];
```

Comment:
This is correct!!! For some more practice:

```javascript

var doubleForLoopArray = [];

for (var i=0; i <2; i++){
	for(var j=0; j<3; j++){
		doubleForLoopArray.push(i+j);
	}
}
```

1. How many elements are going to be in doubleForLoopArray?
2. What will doubleForLoopArray look like?